edit-post/store/reducer.js

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import { get } from 'lodash';
+import mapValues from 'lodash/mapValues';
```

Comment:
I can appreciate the intent in cherry-picking individual modules to avoid bloating the bundle, but as Gutenberg is currently configured, it actually has the opposite effect: it will cause an additional copy of Lodash to be bundled, where one was not previously bundled. See also #5933 and the [Webpack configuration](https://github.com/WordPress/gutenberg/blob/400f8fc685ce309c12ebcb33762c381594002025/webpack.config.js#L164-L165) specifying Lodash as an external.

In fairness, this might not be so clear, so there could be opportunity to improve developer experience here. I could imagine an ESLint rule similar to those we have already [forbidding WordPress imports by name](https://github.com/WordPress/gutenberg/blob/400f8fc685ce309c12ebcb33762c381594002025/.eslintrc.js#L37-L100).